packages/manager/modules/pci/src/projects/project/analytics-data-platform/deploy/deploy.scss

```diff
@@ -0,0 +1,37 @@
+deploy-component {
+  .storage-list {
+    display: flex;
+    flex-wrap: wrap;
+
+    &__term {
+      width: 50%;
+      text-align: left;
+      font-weight: 400;
+    }
+
+    &__definition {
+      width: 50%;
+      overflow: hidden;
+    }
+
+    &__seperator {
+        width: 56%;
+        margin-left: 0;
+    }
+  }
+}
+
+oui-modal {
+  .analytics-data-platform-quota-info {
+    padding-left: 6.7rem !important;
```

Comment:
Don't use `!important`, use more specific selectors :)

Reply:
The `!important` part is remove